roles/rsyslog/tasks/main.yml

```diff
@@ -66,46 +66,72 @@
         path: '{{ __rsyslog_work_dir }}'
         mode: '0700'
 
-    # Back up the pre-existing rsyslog config files in the backup dir,
-    # then removing the files/dirs in rsyslog.d.
+    # Back up the pre-existing rsyslog config files in the backup dir.
+    # If logging_purge_confs is set to true, remove files in rsyslog.d.
+    # Then restore the pre-existing config files.
+    # If logging_reset_confs is set to true, reinstall rsyslog package,
```

Comment:
Where are lines that define what happens when logging_reset_confs == true?

Reply:
https://github.com/nhosoi/logging/blob/restore_confs/roles/rsyslog/tasks/main.yml#L112-L134

Reply:
Ah sorry I am blind, that is fine